content/posts/jellyfin-10-5-0.md

```diff
@@ -0,0 +1,170 @@
+    <td style="text-align:right">0.055 μs</td>
+    <td style="text-align:right">0.051 μs</td>
+    <td style="text-align:right">11.2305</td>
+    <td style="text-align:right">35272 B</td>
+</tr>
+</tbody>
+</table>
+
+Another example of the speed improvements to the server this cycle is the following comparison between our previous alphanumeric comparator and the one shipped in Jellyfin 10.5.0, showing a reduction of the execution time of more than 50%!
+
+<table style="width:100%">
+<thead>
+    <th>Method</th>
+    <th>Mean</th>
+    <th>Error</th>
+    <th>StdDev</th>
+    <th>Gen 0</th>
+    <th>Allocated</th>
+</thead>
+<tbody>
+<tr>
+    <td><b>Jellyfin 10.5.0</b></td>
+    <td style="text-align:right">10.81 μs</td>
+    <td style="text-align:right">0.079 μs</td>
+    <td style="text-align:right">0.074 μs</td>
+    <td style="text-align:right">0.1068</td>
+    <td style="text-align:right">336 B</td>
+</tr>
+<tr>
+    <td><b>Jellyfin 10.4.3</b></td>
+    <td style="text-align:right">23.57 μs</td>
+    <td style="text-align:right">0.055 μs</td>
+    <td style="text-align:right">0.051 μs</td>
+    <td style="text-align:right">11.2305</td>
+    <td style="text-align:right">35272 B</td>
+</tr>
+</tbody>
+</table>
+
+**Support for AMD AMF hardware encoding** is now available on Windows and Linux, as well as support for DVDs as folders, a provider for season images for TheMovieDB and various fixes to media scanning, base URLs and DLNA. **Raspberry Pi hardware acceleration** for older models is also now supported. For Raspeberry Pi 4, hardware-accelerated encoding for H264 is now supported on Raspbian, both using the LinuxServer.io Docker and the repository package.
+
+As part of a project to move the core metadata providers to plugins, MusicBrainz is now a default plugin and allows you to configure the URL of the instance you want to pull data from, thereby allowing you to host an instance of MusicBrainz and sidestep the global rate limiting enforced by the main service.
+
+Aside from all these improvements, more tests have been added on the server-side to help developers track down issues with new and existing code. This ensures a faster development time and prevents some breakages, which will be caught early through automated testing.
+
+## Documentation
+
+Our documentation team has also been hard at work improving both the user and developer documentation.
+
+Of note recently are the overhauls to the [Networking](https://jellyfin.org/docs/general/networking/index.html) and [Hardware Acceleration](https://jellyfin.org/docs/general/administration/hardware-acceleration.html) sections.
+
+We now provide an extensive [Codec Support](https://jellyfin.org/docs/general/clients/codec-support.html) list and help on [CSS Customization](https://jellyfin.org/docs/general/clients/css-customization.html), with examples of useful CSS customizations to apply to your server via the Administration dashboard.
+
+## Looking towards the future
```

Comment:
Although it's great to have all the names listed here, I would add here a *wider* thanks to every user out there, not just the contributors, just like I did in my blog post proposal. See it for reference.